holmscan/cli.py

```diff
@@ -299,6 +331,26 @@ def run(cli_args, sub_args):
                         **tabulate_args
                     )
                 )
+            elif sub_args["show"]:
+                data = c.webscan.get_web_scan(uuid=sub_args["<uuid>"])
+                log.debug(pformat(data))
+                columns = [
+                    "started_date",
+                    "duration",
+                    "status",
+                    "vulnerabilities_count",
+                    "name",
+                ]
+                filtered = []
```

Comment:
List comprehension this

`filtered = [data[column] for column in columns]`

Reply:
Good catch, thanks!